renku/core/commands/dataset.py

```diff
@@ -107,7 +107,7 @@ def create_dataset(client, name, handle_duplicate_fn=None):
     """
     existing = client.load_dataset(name=name)
     if (not existing or handle_duplicate_fn and handle_duplicate_fn(existing)):
-        with client.with_dataset(name=name) as dataset:
+        with client.with_dataset(name=name, create=True) as dataset:
```

Comment:
I think the above line contains unnecessary parentheses. Could we remove it? :smile:

Reply:
If you mean the `if` statement then I did not notice it (since I did not touch it) but I see your point. I think parenthesis is good to have there but the position is wrong; it should be like this:
`if not existing or (handle_duplicate_fn and handle_duplicate_fn(existing)):`